tests/unit-tests/awsconfigs/common/aws-alb-ingress-controller/base/test_data/expected/rbac.authorization.k8s.io_v1_clusterrole_aws-load-balancer-controller-role.yaml

```diff
@@ -0,0 +1,149 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/name: aws-load-balancer-controller
+    kustomize.component: aws-load-balancer-controller
+  name: aws-load-balancer-controller-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - endpoints
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
```

Comment:
are these files coming due to upgrade ?

Reply:
yes these are updates to unit test based on new manifest